src/Cli.Tests/TestHelper.cs

```diff
@@ -1070,6 +1070,7 @@ public static InitOptions CreateBasicInitOptionsForMsSqlWithConfig(string? confi
                 hostMode: HostMode.Development,
                 corsOrigin: new List<string>(),
                 authenticationProvider: EasyAuthType.StaticWebApps.ToString(),
+                restRequestBodyStrict: CliBoolean.True,
```

Comment:
Do we want to explicitly set this? now all tests for CLI will run as if `rest.request-body-strict` was always provided

Reply:
Assigned default value of `CliBoolean.None` which would translate to request-body-strict=true in config json.